edge/pkg/eventbus/event_bus.go

```diff
@@ -35,13 +36,24 @@ type eventbus struct {
 
 func init() {
 	mode := config.CONFIG.GetConfigurationByKey("mqtt.mode")
+	var mqttMode int
+	var ok bool
 	if mode == nil {
-		mode = internalMqttMode
+		mqttMode = internalMqttMode
+	} else {
+		if mqttMode, ok = mode.(int); !ok {
```

Comment:
@nkwangleiGIT 

Are you checking if mqttMode is an integer? Can we simply do something like:

```
i, err := strconv.Atoi("mqttMode")
if err != nil {
  panic("mqtt.mode should be an integer")
}
```

Reply:
The story is that, the original logic get the mqtt.mode from yaml, which the whole yaml content is parsed into
```golang
map[string]interface{}
```
so the underneath mqtt.mode value type is `int` what can cast to int value directly.

However, when deploying using container, we use environment variable to overwrite these config entries, and mqtt.mode parsed into string, so it panics when casting to int.

So adds the conditions to parse mqtt.mode no matter it is a string or int value.

Reply:
Okay, we need some UTs for this change.